index.html

```diff
@@ -0,0 +1,184 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>ICT4D: ICT4D for Development of Africa</title>
+  <link rel="stylesheet" href="./css/styles.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+  <div class="wrapper">
+    <header class="header1">
+      <ul class="little-header-list">
+        <li class="list-item">
+          <a href="#"> <i class="fa fa-facebook"></i> </a>
+        </li>
+        <li class="list-item">
+          <a href="#"> <i class="fa fa-twitter"></i> </a>
+        </li>
+        <li class="list-item">
+          <a href="#"> English </a>
+        </li>
+        <li class="list-item">
+          <a href="#"> My Page </a>
+        </li>
+        <li class="list-item">
+          <a href="#"> Logout </a>
+        </li>
+      </ul>
+
+    </header>
+    <header class="header2">
+      <div class="menu-bar-container">
+        <ul class="menu-bar">
+          <li>
+           <a href="#" class="hamburger">
+            <i class="fa fa-bars menu-bar-icon"></i>
+           </a>
+          </li>
+          <li>
+            <a href="#" class="close-icon"> <i class="fa fa-times"></i> </a>
+          </li>
+        </ul>
+        <div class="menu-image menu-logo-mobile">
+          <a href="./index.html"> <img src="./images/logo.png" alt="Logo Image"> </a>
+        </div>
+      </div>
+      <nav class="navbar">
+        <div class="menu-image menu-image-desk">
+          <a href="./index.html"> <img src="./images/logo.png" alt="Logo Image"> </a>
+        </div>
+        <ul class="menu">
+          <li class="menu-item">
+            <a class="menu-item-link" id="about-link" href="./about.html"> About </a>
+          </li>
+          <li class="menu-item">
+            <a  class="menu-item-link" href="#home"> Home </a>
+          </li>
+          <li class="menu-item">
+            <a class="menu-item-link" href="#agendas"> Program </a>
+          </li>
+          <li class="menu-item">
+            <a class="menu-item-link" href="#speakers"> Speakers </a>
+          </li>
+          <li class="menu-item">
+            <a class="menu-item-link" id="cc-campaign-link" href="#"> CC Campaign </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+  
+    <section id="home"  class="section-conference-invitation">
+      <div class="conference-description-container">
```

Comment:
### Your Homepage Hero Title
![a](https://user-images.githubusercontent.com/81932478/191734564-08b56a90-edd2-45c9-8288-7aa7984757fb.png)

- [ ] Kindly remove this orage background from your homepage and add an h1 title as shown in the design as requested by the previous reviewer :blush:

Reply:
Hi @HAMMAS-SALEEM.  Thank you for your kind feedback.
I have already removed the orange background from the about page. This is not the home page rather it is the front view of the about page which I had removed before I sent my project for the code review. I have already removed the orange background color of the front view of the homepage. However, I see the orange background color of the front view of the homepage has still orange background for the h1 conference title"